File: event-bus-rabbit/src/main/dotnet/topology/events/Registration.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace pegasus.eventbus.topology.events
{
    public class Registration
    {
        public string ClientName;


        public Registration() { }
        public Registration(string clientName) { this.ClientName = clientName; }
    }
}

File: event-bus/event-bus-client/src/main/java/gov/ment/eventbus/topology/events/Registration.java
package gov.ment.eventbus.topology.events;

public class Registration {

  private String clientName;

  // @todo - needed for gson in osgi
  public Registration() {

  }

  public Registration(String clientName) {
    this.clientName = clientName;
  }

  public String getClientName() {
    return clientName;
  }

  public void setClientName(String clientName) {
    this.clientName = clientName;
  }

}
